UI/Web/src/theme/components/_sidenav.scss

```diff
@@ -1,171 +1,236 @@
 @import "../variables";
 
 .side-nav-container {
-    padding-bottom: 10px;
-    width: 190px;
-    background-color: var(--side-nav-bg-color);
-    height: calc((var(--vh) * 100) - 115px);
-    position: fixed;
-    margin: 0;
-    left: 10px;
-    top: 73px;
-    border-radius: var(--side-nav-border-radius);
-    transition:
-        width var(--side-nav-openclose-transition),
-        background-color var(--side-nav-bg-color-transition),
-        border-color var(--side-nav-border-transition);
-    border: var(--side-nav-border);
+  width: var(--side-nav-width);
+  background-color: var(--side-nav-bg-color);
+  height: calc((var(--vh) * 100) - 115px);
+  position: fixed;
+  margin: 0;
+  left: 10px;
+  top: 64px;
+  border-radius: var(--side-nav-border-radius);
+  transition: width var(--side-nav-openclose-transition), background-color var(--side-nav-bg-color-transition), border-color var(--side-nav-border-transition);
+  border: var(--side-nav-border);
+
+  &::-webkit-scrollbar {
+    visibility: hidden;
+  }
+
+  &.preference {
+    height: calc((var(--vh)*100) - 75px);
+    width: 238px;
 
-    &::-webkit-scrollbar {
-        visibility: hidden;
-    }
+    .side-nav {
+      overflow-x: hidden;
 
-    &.preference {
-        height: calc((var(--vh) * 100));
-    }
+      .side-nav-header {
+        color: rgba(255, 255, 255, 0.3);
+        font-size: 15px;
+        &:first-of-type {
+            margin-top: 0.7rem;
+        }
+      }
+
+      .side-nav-item {
+        font-size: 0.9rem;
+        min-height: 30px;
+        padding-left: 12px;
+        justify-content: unset;
+
+        &.active {
+          .active-highlight {
+            width: 4px;
```

Comment:
I'm not sold on the active-highlight changes. Maybe workshop it a bit more, but we should try to keep a consistent ux and with this change the homepage sidebar and preference sidebar would have different behaviors.

Reply:
A rectangle active highlight didn’t make sense to me since it was floating and not flush with the left of the sidebar so again I took inspiration from the Plex settings sidebar and thought that made more sense and also since the settings sidebar and homepage have different uses (homepage tabs are flushed to the left of the sidebar and not indented, along with having card actions)